docs/topics/mta-rn-resolved-issues-6-1-1.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master-6-1-0.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-1_{context}"]
+= Resolved issues
+
+At the time of the release, the following resolved issue has been identified as the major issue worth highlighting:
+
+.RHSSO
+
+The release of {ProductShortName} 6.1.1 resolves the issue that users experienced after the the release of version 7.6.4 of Red Hat Single Sign-On (RH-SSO), which rendered many new installations of {ProductShortName} 6.x unusable.
```

Comment:
@rromannissen & @dymurray please can I ask if you are ok with this draft? I know it is a bit rough, but I am trying to ensure we are not a blocker.

I might change:

* {ProductShortName} *6.x* to *6.X.X*, or do you have a preferred format?
* Change *unusable* to *unstable*

Do you want the ticket number [RHSSO-2514](https://issues.redhat.com/browse/RHSSO-2514) including at the end of the issue?

I have amended the default wording to highlight the singe issue.

Reply:
Thanks @rromannissen